core/src/style/style.h

```diff
@@ -1,22 +1,22 @@
 #pragma once
 
-#include <string>
-#include <vector>
-
 #include "data/tileData.h"
```

Comment:
It depends - when including a struct, like e.g. TileID, does not pull further includes I would not mind to just include it with the header - especially when TileID would need to be included by any user of the header